Chat/package.json

```diff
@@ -17,20 +17,21 @@
     "lint:quiet": "eslint */**/*.{ts,tsx} --quiet"
   },
   "dependencies": {
+    "@azure/communication-calling": "1.4.4",
     "@azure/communication-chat": "1.1.0",
     "@azure/communication-common": "1.1.0",
     "@azure/communication-identity": "1.0.0",
-    "@azure/communication-react": "1.0.0",
-    "@azure/communication-signaling": "1.0.0-beta.5",
+    "@azure/communication-react": "1.2.0",
+    "@azure/communication-signaling": "1.0.0-beta.12",
     "@azure/core-http": "1.2.4",
-    "@fluentui/react": "~8.17.2",
+    "@fluentui/react": "8.66.1",
     "@fluentui/react-icons": "~1.1.137",
     "@types/react-aria-live": "^2.0.0",
     "@types/react-linkify": "^1.0.0",
     "json-stringify-safe": "^5.0.1",
-    "react": "~16.14.0",
+    "react": "18.0.0",
```

Comment:
Our peer deps actually don't allow this:
![image](https://user-images.githubusercontent.com/2684369/164326393-c62122ca-aadd-4aa9-9da7-7d414850dbc8.png)

Purely because react and react-N* both currently require <18 at the moment: https://github.com/microsoft/fluentui/pull/22261

Reply:
That's strange. Chat sample worked perfectly fine after this upgrade :o

Reply:
Changed versions to be inline with peer deps